packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/object/emptyUser/emptyUser.routing.js

```diff
@@ -10,9 +10,14 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       layout: 'modal',
       resolve: {
-        goToUsersAndRoles: /* @ngInject */ ($state) => () =>
-          $state.go('pci.projects.project.users'),
+        goToUsers: /* @ngInject */ ($state) => () => {
+          return $state.go(
+            'pci.projects.project.storages.object-storage.users',
+          );
+        },
```

Comment:
**nitpick**
```suggestion
        goToUsers: /* @ngInject */ ($state) => () => $state.go(
            'pci.projects.project.storages.object-storage.users',
        ),
```

Reply:
QC has been already validated.
And it's another way to complete it.

So no issues with both approach, thanks.